grin/src/miner.rs

```diff
@@ -221,7 +221,7 @@ impl Miner {
 						}
 					}
 				}
-				info!(LOGGER, "Mining at {} graphs per second", sps_total);
+				info!(LOGGER, "Mining! Total Cuckoo power is {} graphs per second", sps_total);
```

Comment:
"power" sounds weird. You want to show Cuckoo on the log line?

Reply:
Yes and yes. Goal is to say "Cuckoo30" to make it hard to miss what algo and gps we're getting (idea triggered by recent test framework bugfix, and some Lobby confusion on benchmarks during in last 5-10 days).

Better wording incoming.